src/store/mutations.ts

```diff
@@ -53,6 +53,60 @@ const mutations: MutationTree<CalChartState> = {
     const currentSS = getSelectedStuntSheet(state);
     currentSS.addDot(dot);
   },
+  syncDotLabelIndices(
+    state,
+    payload: { currentSSDotIndex: number; nextSSDotIndex: number }
+  ): void {
+    // TODO: Design a better way to connect dots between stuntsheets
+
+    // Get the current and next stuntsheets
+    const getSelectedStuntSheet = getters.getSelectedStuntSheet as (
+      state: CalChartState
+    ) => StuntSheet;
+    const currentSS: StuntSheet = getSelectedStuntSheet(state);
+    const getNextStuntSheet = getters.getNextStuntSheet as (
+      state: CalChartState
+    ) => StuntSheet;
+    const nextSS = getNextStuntSheet(state);
+
+    // Set the current dot's label index if not already set
+    const currentDot = currentSS.stuntSheetDots[payload.currentSSDotIndex];
+    if (currentDot.dotLabelIndex === null) {
+      let i: number;
+      for (i = 0; i < currentSS.stuntSheetDots.length; i++) {
+        const dotWithDotLabelIndex = currentSS.stuntSheetDots.findIndex(
+          (dot) => dot.dotLabelIndex === i
+        );
+        if (dotWithDotLabelIndex === -1) {
+          break;
+        }
+      }
+      currentDot.dotLabelIndex = i;
+    } else {
+      // TODO: Can we make this not affect the next ss's dot's pairing with the next next ss?
```

Comment:
Same comment regarding TODOs -- do you think we should have an issue to track this, if this is something we want to do?